.github/workflows/pull-request-label-reviewer.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull Request Label Reviewers
+on:
+  pull_request:
```

Comment:
I think this needs to be `pull_request_target`. `pull_request` doesn't share secrets when run from forks.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Reply:
From the `pull_request` event:

>**Workflows in forked repositories**
Workflows don't run in forked repositories by default. You must enable GitHub Actions in the Actions tab of the forked repository.
> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository. The GITHUB_TOKEN has read-only permissions in pull requests from forked repositories. For more information, see "[Authenticating with the GITHUB_TOKEN](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."

Reply:
Updated